### DISCUSSION TITLE
AX210 Connect to 6GHz question

### DISCUSSION THREAD

**Discussion by Kobunteki** [WLAN-Pi/wlanpi-profiler#154] [Q&A]
Dear experts!

I have heard a lot of issues about AX210's 6Ghz channel accessing  capability under Ubuntu, in that INTEL's driver bans its device accessing 6Ghz channel in countries that does not yet support 6Ghz as Wi-Fi channel.

https://community.intel.com/t5/Wireless/Intel-AX210-Ubuntu-22-04-1-Kernel-5-19-Unable-to-Connect-to-6GHz/td-p/1439765

So friends outside UK, US or other places, have you succeeded in accessing 6Ghz channel using WLAN Pi with AX210 as adapters?

BTW, does MT7921k have the same issue like this?

Thanks experts！

**Comment by MaciejStawiar** [comment]
Yes, here is the solution: https://github.com/WLAN-Pi/feedback/discussions/32
